crates/sui-types/src/messages_consensus.rs

```diff
@@ -161,6 +171,14 @@ pub enum ConsensusTransactionKind {
     CapabilityNotification(AuthorityCapabilities),
     NewJWKFetched(AuthorityName, JwkId, JWK),
     RandomnessStateUpdate(u64, Vec<u8>),
+    // DKG is used to generate keys for use in the random beacon protocol.
+    // `RandomnessDkgMessage` is sent out at start-of-epoch to initiate the process.
+    // Contents are a serialized `fastcrypto_tbls::dkg::Message`.
+    RandomnessDkgMessage(AuthorityName, Vec<u8>),
+    // `RandomnessDkgConfirmation` is the second DKG message, sent as soon as a threshold amount of
+    // `RandomnessDkgMessages` have been received locally, to complete the key generation process.
+    // Contents are a serialized `fastcrypto_tbls::dkg::Confirmation`.
```

Comment:
why do we have double serialization here?

Reply:
this was copied over from Narwhal, where we did it to avoid needing to always deserialize DKG messages in all the places where a header was deserialized. not as much of a concern here so i will undo it